src/toucan/models.clj

```diff
@@ -371,6 +373,45 @@
 ;;;                                                      DEFMODEL MACRO
 ;;; ========================================================================================================================
 
+(defn- ifn-invoke-forms
+  "Macro helper, generates
+
+       (~'invoke [this#]
+        (invoke-model-or-instance this#))
+       (~'invoke [this# id#]
+        (invoke-model-or-instance this# id#))
+       (~'invoke [this# arg1# arg2#]
+        (invoke-model-or-instance this# arg1# arg2#))
+       ,,,"
```

Comment:
`,,,` is called a "fogus comma", it's some used in Clojure docs (e.g. [this one](https://clojure.org/guides/threading_macros), as well by [PragProg](https://media.pragprog.com/titles/vmclojeco/intro.pdf)

![2017-06-01-145244_615x93_scrot](https://cloud.githubusercontent.com/assets/32212/26680510/00fef6d2-46da-11e7-846e-ad0598f6f825.png)

I can change it though if you don't like it.